topics/proteomics/tutorials/multiGSEA-tutorial/tutorial.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: tutorial_hands_on
+
+title: Using MultiGSEA
+subtopic: multi-omics
+tags: 
+  - multi-omics
+  - transcriptomics
+  - proteomics
+  - metabolomics
+zenodo_link: 'https://zenodo.org/records/14216972'
+questions:
+- How to use MultiGSEA for GSEA-based pathway enrichment for multiple omics layers?
+objectives:
+- Perform GSEA-based pathway enrichment for transcriptomics, proteomics, and metabolomics data.
+- Understand how to combine p-values across multiple omics layers.
+time_estimation: 1H
+key_points:
+- MultiGSEA provides an integrated workflow for pathway enrichment analysis across multi-omics data.
+- Supports pathway definitions from several databases and robust ID mapping.
+contributors:
+- tStehling
+
+
+---
+
+
+The multiGSEA package was designed to run a robust GSEA-based pathway enrichment for multiple omics layers. The enrichment is calculated for each omics layer separately and aggregated p-values are calculated afterwards to derive a composite multi-omics pathway enrichment.
+
+Pathway definitions can be downloaded from up to eight different pathway databases by means of the graphite Bioconductor package (Sales, Calura, and Romualdi 2018). Feature mapping for transcripts and proteins is supported towards Entrez Gene IDs, Uniprot, Gene Symbol, RefSeq, and Ensembl IDs. The mapping is accomplished through the AnnotationDbi package (Pagès et al. 2019) and currently supported for 11 different model organisms including human, mouse, and rat. ID conversion of metabolite features to Comptox Dashboard IDs (DTXCID, DTXSID), CAS-numbers, Pubchem IDs (CID), HMDB, KEGG, ChEBI, Drugbank IDs, or common metabolite names is accomplished through the AnnotationHub package metabliteIDmapping. This package provides a comprehensive ID mapping for more than 1.1 million entries.
+
+This tutorial covers a simple example workflow illustrating how the multiGSEA package works. The omics data sets that will be used throughout the example were originally provided by Quiros et al. (Quirós et al. 2017). In their publication the authors analyzed the mitochondrial response to four different toxicants, including Actinonin, Diclofenac, FCCB, and Mito-Block (MB), within the transcriptome, proteome, and metabolome layer.
+In this tutorial we will solely focus on the Actinonin data set.
+
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Preparing the Data
+
+To perform pathway enrichment with MultiGSEA, you'll need omics datasets in the file type TSV . Each individual data set contains four columns representing the feature (denoted as Symbol), the log2 fold change (logFC), the p-value (pValue), and the adjusted p-values (adj.pValue). We'll use example data provided on Zenodo.
```

Comment:
you have provided the technical description of the dataset, is there also a biological description that can be given? what samples are in these files? what was the research question that led to this data, etc?

Reply:
ok, I see now that you did that already a bit in the text above, so please disregard my comment